0-bootstrap/outputs.tf

```diff
@@ -77,6 +77,16 @@ output "gcs_bucket_cloudbuild_artifacts" {
   value       = { for key, value in module.tf_workspace : key => replace(value.artifacts_bucket, local.bucket_self_link_prefix, "") }
 }
 
+output "projects_gcs_bucket_tfstate" {
+  description = "Bucket used for storing terraform state for stage 4-projects foundations pipelines in seed project."
+  value       = local.projects_gcs_bucket_tfstate
+}
+
+output "cloud_builder_artifact_repo" {
```

Comment:
nit: let's open an issue on builder to expose full self link

Reply:
Issue created https://github.com/terraform-google-modules/terraform-google-bootstrap/issues/199